test/universal.js

```diff
@@ -8,11 +8,12 @@ var GoogleAnalytics = require('..');
 var Test = require('segmentio-integration-tester');
 var assert = require('assert');
 var each = require('lodash.foreach');
-var enhancedEcommerceMethods = require('../lib/universal/enhanced-ecommerce');
 var fmt = require('util').format;
-var helpers = require('./helpers');
+
+var enhancedEcommerceMethods = require('../lib/universal/enhanced-ecommerce');
```

Comment:
just curious, is breaking apart 3rd party module deps and relative `require`s enforced by our lint config?